### PR TITLE
Some missing manager->pilot and getting presubmit to work again

### PR DESCRIPTION
**Pull request by ldemailly**
leftovers/breaking changes from #860 and #825 and #863

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -332,7 +332,7 @@ DEBUG_BASE_IMAGE_SHA = "79702c8cc8c89117b99a2ccf591d5b9ddb4708eb3cf7806ceb3e2d61
 http_file(
     name = "ubuntu_xenial_debug",
     sha256 = DEBUG_BASE_IMAGE_SHA,
-    url = "https://storage.googleapis.com/istio-build/manager/ubuntu_xenial_debug-" + DEBUG_BASE_IMAGE_SHA + ".tar.gz",
+    url = "https://storage.googleapis.com/istio-build/pilot/ubuntu_xenial_debug-" + DEBUG_BASE_IMAGE_SHA + ".tar.gz",
```
Comment (kyessenov) [inline WORKSPACE]: have you pushed the image manually?

**Comment by ldemailly** [comment]
I don't have access to push to istio-build/pilot - @sebastienvas  can you add me/tell me how ?
```
gsutil cp ubuntu_xenial_debug.tar.gz gs://istio-build/pilot/ubuntu_xenial_debug-b5da9ebb9dfc0dcb2c15708017ae6799139279bfe7b76e11d98ca4a054b77991.tar.gz
```
AccessDeniedException: 403 Caller does not have storage.objects.list access to bucket istio-build.

**Comment by sebastienvas** [comment]
Try it again now. I just changed the IAM permission

**Comment by ldemailly** [comment]
@sebastienvas 
I can't still getting 
```
gsutil ls gs://istio-build/pilot/
AccessDeniedException: 403 Caller does not have storage.objects.list access to bucket istio-build.
```
will bug you tomorrow

**Comment by ldemailly** [comment]
same from internal vm - I think I need to use the expected credentials or you need to add my account (my internal email)

**Comment by sebastienvas** [comment]
Yeah I think you cannot list only read and write.

File: WORKSPACE (hunk 1 above)
Comment (ldemailly) [inline WORKSPACE]: done. PTAL

**Comment by kyessenov** [comment]
Please make sure tests pods locally. Jenkins is still broken

**Comment by sebastienvas** [comment]
@kyessenov Jenkins was not actually broken, the istioctl cross compile script was being called and that is the reason the build failed. Laurent remove the call in Jenkinsfile, and now it is actually starting pilot e2e.

**Comment by ldemailly** [comment]
@kyessenov 
```
+ bin/codecov.sh
# istio.io/pilot/tools/version
import cycle not allowed in test
package istio.io/pilot/tools/version (test)
	imports istio.io/pilot/tools/version

FAIL	istio.io/pilot/tools/version [setup failed]
```

**Comment by ldemailly** [comment]
@kyessenov now that the build is finally fixed, after 6 days, I hope you see the value in keeping it working as several of your PRs were breaking and forced merged/undetected

File: tools/version/version_test.go [hunk 2]
```diff
@@ -14,14 +14,10 @@
 
 package version
 
-import (
-	"testing"
-
-	"istio.io/pilot/tools/version"
-)
+import "testing"
```
Comment (kyessenov) [inline tools/version/version_test.go]: should be package version_test but this works too. a bad merge...

**Pull request merged**